byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -52,4 +72,21 @@ private String formatLocation(PlaceType location) {
     place = place.replace('_', ' ');
     return place;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String tripKeyString = request.getParameter("tripId");
+    Key tripKey = KeyFactory.stringToKey(tripKeyString);
+    ArrayList<String> interests = gson.fromJson(request.getReader(), ARRAYLIST_STRING);
+    Entity trip;
+    try {
+      trip = datastore.get(tripKey);
+      trip.setProperty("interests", interests);
+      datastore.put(trip);
+      response.setContentType("application/json;");
+      response.getWriter().println("{\"results\": \"success\"}");
```

Comment:
Do you need to return this "success" value here? The frontend should know whether the operation was successful based on the return code of the response. Unless you're returning something meaningful, you probably don't need to this.

Reply:
Updated